js/form-disable.js

```diff
@@ -1,5 +1,6 @@
 var $ = require("jquery");
 
-$(".preview form").submit(function (e) {
-  e.preventDefault();
+$(".site-component-preview form").on("submit", (e) => {
+  e.preventDefault;
+  return false;
```

Comment:
Prevents forms being submitted in component previews, like [Footer](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-oct-23-update-dependencies/components/footer/).

<img width="600" alt="image" src="https://github.com/uswds/uswds-site/assets/3385219/6869a7d0-64d6-455d-b66e-e11ebc671175">